apps/browser/pages/api/agent-jobs/start.ts

```diff
@@ -0,0 +1,27 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { evoAgentJobScheduler } from "../../../src/agent-jobs/AgentJobScheduler";
```

Comment:
For backend utility functions that should not be run within the frontend, we've been putting them outside of the `src/` directory as that is assumed to be the root directory of your web app.

See here, where we've created an `api-utils` directory: https://github.com/polywrap/evo.ninja/tree/dev/apps/browser/api-utils